src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -841,15 +948,15 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                             $posmsg += ""`n"" + $indentString
```

Comment:
Can we use single quotes too?

Reply:
Same as above, single quote will treat backtick as literal so it won't be a new line.